js/Domupdates.js

```diff
@@ -0,0 +1,87 @@
+// start with event listner for start button that triggers the start of the game, dommanipulation method for putting names on the screen, generates the puzzle and populates it on the screen. Intialize game function.
+
+const player1 = document.querySelector('.player1-name-input');
+const player2 = document.querySelector('.player2-name-input');
+const player3 = document.querySelector('.player3-name-input');
+const startGameButton = document.querySelector('.start-button');
+const startScreen = document.querySelector('.start-screen');
+const player1Name = document.querySelector('.player1-name');
+const player2Name = document.querySelector('.player2-name');
+const player3Name = document.querySelector('.player3-name');
+
+startGameButton.addEventListener('click', startGame);
+
+let game;
+let puzzle;
+
+function startGame() {
+  game = new Game(player1.value, player2.value, player3.value);
+  startScreen.classList.add('hidden');
+  populatePlayerNames()
+}
+
+const resetGameBtn = document.querySelector('.new-game-btn');
+resetGameBtn.addEventListener('click', resetGame);
+
+function resetGame() {
+  
+}
+
+function populatePlayerNames() {
+  player1Name.innerText = player1.value;
+  player2Name.innerText = player2.value;
+  player3Name.innerText = player3.value;
+}
+
+const wheelButton = document.querySelector('.wheel-btn');
+wheelButton.addEventListener('click', showSpinValue);
+const spinValue = document.querySelector('.current-spin-value');
+
+function showSpinValue() {
+  spinValue.innerText = 10;
+}
+
+const guessLetterInput = document.querySelector('.letter-guess-input');
+const letterGuessSubmitButton = document.querySelector('.letter-guess-submit-btn');
+letterGuessSubmitButton.addEventListener('click', showLetterGuessed);
+const letterA = document.querySelector('.letter-a');
+const letterB = document.querySelector('.letter-b');
+const letterC = document.querySelector('.letter-c');
+const letterD = document.querySelector('.letter-d');
+const letterE = document.querySelector('.letter-e');
+const letterF = document.querySelector('.letter-f');
+const letterG = document.querySelector('.letter-g');
+const letterH = document.querySelector('.letter-h');
+const letterI = document.querySelector('.letter-i');
+const letterJ = document.querySelector('.letter-j');
+const letterK = document.querySelector('.letter-k');
+const letterL = document.querySelector('.letter-l');
+
+function showLetterGuessed(e) {
```

Comment:
I really like the logic here. I wonder if this type of thing should live in the domUpdates file? Might be a good question to ask the instructors.